backend/services/syson-services/src/main/java/org/eclipse/syson/services/UtilService.java

```diff
@@ -142,6 +146,32 @@ public List<EObject> getAllReachable(EObject eObject, EClass eClass) {
         return allReachable;
     }
 
+    /**
+     * Find an {@link Element} that match the given name in the ResourceSet of the given element.
```

Comment:
This comment seems obsolete. Please be sure that `namespace.resolve(elementName);` search in the whole resource set and fix the comment if needed.

Reply:
`namespace.resolve` is global to the whole ResourceSet.